### ISSUE TITLE
Canada Post/Postes Canada `amenity=post_box` entries should be merged

### ISSUE THREAD

**Issue by wolfy1339** [osmlab/name-suggestion-index#6321]
The current suggestion to have a different tagging for Quebec and the rest of Canada has been replaced with a harmonized bilingual approach. The consensus agreed on the talk-ca mailing list is as follows:

```
amenity=post_box
operator:en=Canada Post
operator:fr=Postes Canada
operator:wikidata=Q1032001
```

Ref: https://lists.openstreetmap.org/pipermail/talk-ca/2021-September/010097.html

**Comment by arch0345** [comment]
Reading through that thread, it looks like some people suggested removing `operator=*` in favor of just `operator:en=*` and `operator:fr=*` while others were fine with leaving the French `operator` in Quebec and the English `operator` in the rest of Canada in addition to the multilingual operator tags. Since NSI entries in the operators tree require an `operator` tag, I think it would make more sense to add the multilingual operators to both entries. Merging these could cause a conflict of what to put as the `displayName` (these currently aren't translatable: #5982)  and `operator`

**Comment by EOEboh** [comment]
Then I'd suggest a separate JSON list for Quebec Canada and some other bilingual countries

**Comment by une-abeille-osm** [comment]
I agree that the entries for these post boxes should be merged for the whole country, because the boxes are fully bilingual. Because they display both names, it does not harm a user's ability to find the box if their device displays it in English while they are in Québec, or vice versa.

https://en.wikipedia.org/wiki/File:Canadapost_mailbox.jpg
